pkg/csconfig/database.go

```diff
@@ -131,6 +131,9 @@ func (d *DatabaseCfg) ConnectionString() string {
 		} else {
 			connString = fmt.Sprintf("%s:%s@tcp(%s:%d)/%s?parseTime=True", d.User, d.Password, d.Host, d.Port, d.DbName)
 		}
+		if d.Sslmode != "" {
```

Comment:
actually, this can be "require" or "disable", right?

Reply:
for mysql its, true, false, "skip-verify", "required" but its just a string, we detect if its not empty